### PR TITLE
feat: track & comment bundle size change

### PR DESCRIPTION
**Pull request by sakulstra**

### CODE REVIEW

File: scripts/compare-size.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    const currentMainChunk = stats.results.find((chunk) => /js\/main/.test(chunk.bundleName));
+    const currentMB = currentMainChunk.totalBytes / 1024 / 1024;
+
+    const diff = currentMB - masterMB;
+    await github.rest.issues.createComment({
+      issue_number: context.issue.number,
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+      body: `Bundle size change: ${diff}MB`,
```
Comment (sakulstra) [inline scripts/compare-size.js:17]:
```suggestion
      body: `Bundle size change: ${diff} MB`,
```

**Review by foodaka — DISMISSED** [review]
Nice stuff

**Comment by foodaka** [comment]
@sakulstra do you wanna add some documentation on how it works?

**Review by foodaka — APPROVED** [review]
thanks for adding the docs, looks good

**Pull request merged**